doc/ping.xml

```diff
@@ -851,6 +851,28 @@ xml:id="man.ping">
     hosts).</para>
   </refsection>
 
+  <refsection xml:id="id_collisions">
+    <info>
+      <title>ID COLLISIONS</title>
+    </info>
+    <para>
+    Unlike TCP and UDP, which use port to uniquely identify the socket to
+    deliver data, ICMP use identifier field (ID) to identify the socket.
+    Therefore if on the same machine, at the same time, two ping processes
+    use the same ID, echo reply can be delivered to the wrong socket.
+    This is known problem due 16 bit ID field (65535).
+    <command>ping</command> prints <emphasis remap="I">DIFFERENT ADDRESS</emphasis>
+    error and packet loss is negative.
+    </para>
+    <para>
+    <command>ping</command> uses PID to get unique number.  The default value of
+    <emphasis remap="I">/proc/sys/kernel/pid_max</emphasis> is 32768.
+    On the systems that use ping heavily it's important to have
+    <emphasis remap="I">/proc/sys/kernel/pid_max </emphasis>
+    value less or equal to 65535.
```

Comment:
Maybe something along the lines 'on the systems that use ping heavily and with pid_max greater than 65535 collisions are bound to happen' and maybe add that this is a historical limitation of the protocol that we cannot really fix at this point. Also if we have an option to encode an ID into the ping packet payload we should probably mention it here as well.